subqueries/coreum-testnet/src/readme.md

```diff
@@ -0,0 +1,26 @@
+# Types Conflict Issue - Notice and Solution
+
+## Issue Description
+When running the indexer, an issue arises due to conflicts in the types of `MsgMint` between FT (Fungible Token) and NFT (Non-Fungible Token) within the `CosmosMessageTypes.ts` file. These message types lack differentiation, causing conflict during code generation or compilation.
```

Comment:
Please add a complete path name like this `./src/types/CosmosMessagTypes.ts`

Reply:
can you check now please